setup.py

```diff
@@ -49,6 +49,7 @@
         "Jinja2 >= 2.11", # I need "include" and auto-context + blank line are not indented by default
         "pyyaml",
         "mistune < 2.0.0",  # Need to pin mistune's max version so m2r doesn't break
+        "docutils == 0.18.1", # Need to pin docutils on 0.18.1 so m2r doesn't break
```

Comment:
@changlong-liu sorry for the confusion yesterday, what we want is to keep the requirements.txt and setup.py the same, but update the dev_requirements.txt to get the pipelines workign again